tests/paths.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2023 The MobileCoin Foundation
+
+from filecmp import dircmp
+from pathlib import Path
+import seedir as sd
+
+#: The root of this repository. This can be used as the `template` parameter to
+#: `cookiecutter` specifying the directory to the desired template as one would
+#: do on the command line.
+TEMPLATE_ROOT = Path(__file__).parent.parent
+
+
+class ReprPath:
```

Comment:
Up for suggestions on the names of these classes, and it could be I'm putting classes where they aren't needed.

Reply:
Removed the ReprPath, non longer needed